modules/end-user-guide/partials/ref_devfile-reference.adoc

```diff
@@ -901,7 +901,7 @@ components:
       app: postgres
 ----
 
-Additionally, it is also possible to modify the entrypoints (command and arguments) of the containers present in the resource list. 
+Additionally, it is also possible to modify the entrypoints (command and arguments) of the containers in the resource list. 
```

Comment:
entry points?
```suggestion
Additionally, you can modify the entrypoint commands and arguments of the containers in the resource list. 
```

Reply:
@MichalMaler entrypoints is used in the procedure as one word, so I assume that is correct. However, sometimes it is capitalized and sometimes it isn't, is that intended? I don't know enough about it to decide, if we can safely remove the brackets without altering the meaning. Can you please take a look?

Reply:
will do!

Reply:
Regarding what I got from developers, there is no need to capitalize this. Then, the correct expression coming from the Kubernetes slang is `entrypoint`. On top of that, we are talking about something from the devfile reference, so it is not to be mentioned as `entry point`.